src/scrab-bag.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+export default function ScrabBag(input) {
+  let scrabbleLetters = {
```

Comment:
This is not the list of tiles and numbers that you should be using. See [this table](http://scrabblewizard.com/scrabble-tile-distribution/).